### PR TITLE
[WindowsIotServices]First preview release.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

First preview release.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by wyunchi-ms — COMMENTED** [review]
Please also modify `documentation/azure-powershell-modules.md`

**Comment by LucasYao93** [comment]
Have updated.

**Comment by LucasYao93** [comment]
> Have updated.

**Comment by LucasYao93** [comment]
> Please also modify `documentation/azure-powershell-modules.md`

Have updated.

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/WindowsIotServices/Changelog.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Upcoming Release
+
+## Version 0.1.0
```
[the remark this answers is not in the document]
Comment (LucasYao93) [reply src/WindowsIotServices/Changelog.md:22]: We need to add  change log in Changelog.md ? so why remove it ?

[the remark this answers is not in the document]
Comment (dingmeng-xue) [reply src/WindowsIotServices/Changelog.md:22]: For the first release, it is fine.

**Pull request merged**